src/main/java/org/apache/sysml/runtime/matrix/data/LibMatrixDatagen.java

```diff
@@ -510,14 +511,19 @@ private static void genRandomNumbers(boolean invokedFromCP, int rl, int ru, int
 
 		// Setup Pseudo Random Number Generator for cell values based on 'pdf'.
 		if (valuePRNG == null) {
-			if ( rgen._pdf.equalsIgnoreCase(RAND_PDF_UNIFORM)) 
+			switch (rgen._pdf) {
+			case UNIFORM:
```

Comment:
I am going to leave this the way the now committed style format requires me to. If the future discussion about the style template produces a different outcome, I shall change it to that.